client-http/src/main/java/com/cloudera/livy/client/http/HttpConf.java

```diff
@@ -80,4 +85,51 @@ private Entry(String key, Object dflt) {
   boolean isSpnegoEnabled() {
     return getBoolean(Entry.SPNEGO_ENABLED);
   }
+
+  private static final Map<String, DeprecatedConf> configsWithAlternatives
+    = new HashMap<String, DeprecatedConf>() {{
+      put(HttpConf.Entry.JOB_INITIAL_POLL_INTERVAL.key, DepConf.JOB_INITIAL_POLL_INTERVAL);
+      put(HttpConf.Entry.JOB_MAX_POLL_INTERVAL.key, DepConf.JOB_MAX_POLL_INTERVAL);
+  }};
+
+  // Maps deprecated key to DeprecatedConf with the same key.
+  // There are no deprecated configs without alternatives currently.
+  private static final Map<String, DeprecatedConf> deprecatedConfigs = new HashMap<>();
+
+  public Map<String, DeprecatedConf> getConfigsWithAlternatives() {
+    return Collections.unmodifiableMap(configsWithAlternatives);
```

Comment:
Thanks, I'll change this to `protected`and move the wrap to the map declaration (on all the files)